server/src/main/java/com/cloud/uuididentity/UUIDManagerImpl.java

```diff
@@ -119,7 +119,9 @@ public boolean IsUuidFormat(String uuid) {
         }
         Identity identity = (Identity) this._entityMgr.findById(entityType, customId);
         if (identity == null) {
-            throw new InvalidParameterValueException("Unable to find UUID for id " + customId);
+            throw new InvalidParameterValueException(String.format("Unable to find UUID for id %s of type %s",
+                                                                   customId,
+                                                                   (entityType == null ? "<unknown type>" ? entityType.getSimpleName())));
```

Comment:
```suggestion
                                                                   (entityType == null ? "<null>" ? entityType.getSimpleName())));
```

Reply:
I am open to alternatives, but I don't like the <null>, how about just "unknown"?

Reply:
"unknown" sounds good.

Reply:
+1